torch/csrc/jit/constants.cpp

```diff
@@ -16,6 +16,32 @@ c10::OperatorOptions aliasAnalysisInternalSpecialCase() {
 }
 } // namespace
 
+bool insertableTensor(const at::Tensor& ten) {
```

Comment:
Shouldn't the [`val.isTensor`](https://github.com/pytorch/pytorch/pull/31841/files?file-filters%5B%5D=.cpp&file-filters%5B%5D=.h#diff-a28bcf9c5b0581cf3dd1179580c2f957R65) case for `tryInsertConstant` also have this check?

Reply:
maybe, but i think the stack trace would be worse in that case.